src/type/directives.ts

```diff
@@ -248,11 +248,12 @@ export const GraphQLSpecifiedByDirective: GraphQLDirective =
   });
 
 /**
- * Used to declare an Input Object as a OneOf Input Objects.
+ * Used to indicate an Input Object is a OneOf Input Object.
  */
 export const GraphQLOneOfDirective: GraphQLDirective = new GraphQLDirective({
   name: 'oneOf',
-  description: 'Indicates an Input Object is a OneOf Input Object.',
+  description:
+    'Indicates that exactly one field must be supplied and that field must not be `null`.',
```

Comment:
```suggestion
    'Indicates that exactly one field must be supplied and that the field must not be `null`.',
```

Reply:
I don't think so. Arguably "and that _that field_ must not be `null`" would work, but generally it's best to remove redundant "that"'s. I guess `and this` would also work:
```suggestion
    'Indicates that exactly one field must be supplied and that this field must not be `null`.',
```